examples/testify/src/integration-test/groovy/gorm/tools/SanityCheckSpec.groovy

```diff
@@ -16,4 +18,22 @@ class SanityCheckSpec extends Specification implements DataIntegrationTest {
         KitchenSink.load(2)
     }
 
+    void "check isDirty throws error"() {
```

Comment:
can we get a series in IsDirtySpec in gorm-tools? does it need to be integrationTest? what works and what does not? can we fix it? what do we do and what do we use? this isn't telling us to much.